components/ILIAS/Forum/PRIVACY.md

```diff
@@ -0,0 +1,99 @@
+If **statistics** are activated in the forum, a table is available for all users with read access, in which at least the **username** and the **number of posts** are listed for users with posts. If the personal profile is enabled, the first name and surname are also displayed. If a **learning progress** is activated in the forum and you have the authorization to see the learning progress of other users, it will be displayed here.
+
+If the **notification settings** of a forum allow the individal deactivation of an automatic notification, a list of all course or group members with username, first and last name, as well as the permission is displayed.
+
+## Data being deleted
+
+When a **post or topic** is deleted, all associated personal data is deleted. If a daily notification via cron job is set in the ILIAS administration, this data is stored temporarily until the notifications are sent.
+
+After deleting a **user account**, the user's topics and posts remain in the forum. Their data records still contain the ID and, in the case of non-pseudonymized posts, also the username. However, "Deleted" is displayed as the name of the creator. The user account no longer appears in the forum statistics.
+
+If an **entire forum** is deleted and the ILIAS "Trash" is active, all personal data is initially retained.  When the forum object is finally deleted from "Trash", all forum data (incl. user related data) is deleted as well.
+
+## Data being exported
+
+If a **notification about new or changed posts** is set, it will be sent via the ILIAS mail system. For users who have set forwarding to an external e-mail address, the notification will be forwarded to this address. The notification contains the date, title and content of the post, as well as the username or pseudonym of the creator. Depending on the setting, file attachments are also forwarded.
+
+New posts can be subscribed to via an **RSS feed** in suitable programs. In the forum, you can set whether this feed is password-protected or public. A public feed contains the title of the forum, the **title, content, creation date** and a link to the post in ILIAS for each post. If the forum itself is not public, you will be taken to the ILIAS login page when you use the link.
+
+To support the **full-text search**, a separate server with the Apache Lucene search engine can be operated for ILIAS. When creating the full-text index for a forum, the content of a manually created start page as well as the title, content and author ID of all topics and posts are retrieved by this server and stored in the index.
+
+ILIAS users can create a **print view** in a forum topic and print it out via the browser. The personal data contained on it corresponds to the normal browser view of the forum topic. ILIAS users with the 'Edit Settings' permission for a forum can export an **HTML page** with the content of all forum topics. The personal data contained in it corresponds to the normal browser views of the forum topics.
```

Comment:
```suggestion
ILIAS users can create a **print view** in a forum topic and print it out via the browser. The personal data contained on it corresponds to the normal browser view of the forum topic. ILIAS users with the "Edit Settings" permission for a forum can export an **HTML page** with the content of all forum topics. The personal data contained in it corresponds to the normal browser views of the forum topics.
```
according to How to write the privacy.md, "" shold be used to indicate permissions